challenge-01/README.md

```diff
@@ -2,41 +2,44 @@
 
 ```js
 // Declarar uma variável chamada `myvar`, sem valor.
-?
+node // para chamar o nodejs caso estejamos trabalhando com ele
+var myvar;
 
 // Após declarada, atribua o valor 10 à variável `myvar`.
-?
+myvar = 10;
 
 // Declare uma nova variável chamada `soma`, e adicione uma instrução somando os valores 15 e 8.
-?
+var soma = 15 + 8;
 
 // Atribua à variável `soma` todo o valor dela, somando 1, usando o operador de soma abreviado.
-?
+++soma;
 
 // Atribua à variável `soma` todo o valor dela, multiplicando por 3, usando o operador de multiplicação abreviado.
-?
+soma * 3;
 
 // Qual é o valor da variável `soma` até aqui?
-?
+72;
 
 // Declare uma variável chamada `souninja`, atribuindo à ela o valor booleano que representa `verdadeiro`.
-?
+var souninja = true;
 
 // Declare uma variável chamada `comida` que recebe um array com os valores 'arroz', 'feijão' e 'ovo'.
-?
+var comida = ['arroz','feijao', 'ovo']
 
 // Digite a instrução que imprime o valor de 'feijao', que está na variável `comida`.
-?
+comida[1]
 
 // Digite o código que verifica se a variável `soma' é igual a variável `myvar` (testando também o tipo).
-?
+soma === myvar
 
 // Digite o código que verifica se a variável `myvar` é menor ou igual à variável `soma`.
-?
+myvar <= soma;
 
-// Crie uma função chamada `divisao` que receba como parâmetro dois números, e retorne o resultado da divisão entre eles.
-?
 
+// Crie uma função chamada `divisao` que receba como parâmetro dois números, e retorne o resultado da divisão entre eles.
+function divisao(){
+... x = 10 / 2;
+... return x;
+... }
```

Comment:
Essa função não funcionaria, pois os `...` adicionados seriam interpretados de forma errada pela engine do JS. O console do `node` adiciona esses pontos só pra você saber que ali você pode continuar a instrução. Não adicione eles quando for digitar o código.